### PR TITLE
[REF] point_of_sale: split pos tour assets in folders

### PR DESCRIPTION
**Pull request by shag-odoo**
Following this commit:
In the test files of the point_of_sale module there are many helper functions
that are used throught the pos modules. The problem is that the generic helper
functions are coupled with the regular pos tour helpers. 

As such it is difficult to import  the desired helpers. What we have to resort
to is importing the whole pos tours folder wherever we want to use said helpers.

Here we restructure the `pos/static/tests` folder in order to solve
this issue.

New structure:
/pos/static/tests
   - /generic_helpers
   - /customer_display
   - /pos

task- 4240939
https://github.com/odoo/enterprise/pull/72002

### CODE REVIEW

**Review by clbr-odoo — APPROVED** [review]

**Comment by vlst-odoo** [comment]
Thanks for this @shag-odoo . 

Can you also move some of the files from `pos/tours/utils` to `tests/generic_components_helpers/`, for ex:
```
dialog_util.js
number_popup_util.js
numpad_util.js
selection_popup_util.js
text_input_popup_util.js
```
and also some of the methods from `common.js` can be moved to a `utils.js` file inside `generic_components_helpers`, for ex:
```
negate(selector, parent = "body")
negateStep(step)run(run, content = "run function")
refresh()
elementDoesNotExist(selector)

```

Since the `generic_components_helpers` folder doesn't only contain "component's" helpers it might make sense to rename it to smth like `generic_helpers`

**Comment by shag-odoo** [comment]
Hello @vlst-odoo 
Changes have been done
Could you please have a look

Thanks and have a great day :innocent:

File: addons/point_of_sale/static/tests/generic_helpers/order_widget_util.js [hunk 1]
```diff
@@ -98,3 +101,19 @@ export function hasTax(amount) {
...
+}
+export function refresh() {
+    return run(() => window.location.reload(), "refresh page");
+}
+export function elementDoesNotExist(selector) {
+    return {
+        content: `Check that element "${selector}" don't exist.`,
+        trigger: negate(selector),
+    };
+}
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/tests/generic_helpers/order_widget_util.js:119]: these functions should be in a separate file. ex: `/generic_helpers/utils.js`

Comment (shag-odoo) [reply addons/point_of_sale/static/tests/generic_helpers/order_widget_util.js]: It is been updated according to the above mention folder structure.
Thanks!

File: addons/l10n_test_pos_qr_payment/static/tests/test_pos_payment_tour.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
-import * as PaymentScreen from "@point_of_sale/../tests/tours/utils/payment_screen_util";
-import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
-import * as Chrome from "@point_of_sale/../tests/tours/utils/chrome_util";
+/** @odoo-module */
+
```
Comment (vlst-odoo) [inline addons/l10n_test_pos_qr_payment/static/tests/test_pos_payment_tour.js:2]:
```suggestion
```

**Comment by shag-odoo** [comment]
Hello @vlst-odoo 
Changes have been done, Could please have a look over it.

Thanks and have a great day ahead :innocent:

File: addons/point_of_sale/static/tests/generic_helpers/utils.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { run } from "@point_of_sale/../tests/pos/tours/utils/common";
```
[the remark this answers is not in the document]
Comment (shag-odoo) [reply addons/point_of_sale/static/tests/generic_helpers/utils.js]: Updated !
Thanks!

File: addons/pos_viva_wallet/static/tests/tours/viva_wallet_tour.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
-import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
-import * as PaymentScreenPos from "@point_of_sale/../tests/tours/utils/payment_screen_util";
+/** @odoo-module */
+
```
Comment (vlst-odoo) [inline addons/pos_viva_wallet/static/tests/tours/viva_wallet_tour.js:2]: you can remove these lines

File: addons/l10n_be_pos_sale/static/tests/tours/l10n_be_pos_sale_tour.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
-import * as PaymentScreen from "@point_of_sale/../tests/tours/utils/payment_screen_util";
-import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
-import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
+/** @odoo-module */
```
Comment (vlst-odoo) [inline addons/l10n_be_pos_sale/static/tests/tours/l10n_be_pos_sale_tour.js:1]: remove this

File: addons/point_of_sale/static/tests/pos/tours/utils/product_screen_partner_list_util.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+/** @odoo-module */
+
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/tests/pos/tours/utils/product_screen_partner_list_util.js:2]: remove this

File: addons/pos_sale/static/tests/tours/utils/pos_sale_utils.js [hunk 7]
```diff
@@ -1,6 +1,8 @@
-import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
-import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
-import * as Numpad from "@point_of_sale/../tests/tours/utils/numpad_util";
+/** @odoo-module */
+
```
Comment (vlst-odoo) [inline addons/pos_sale/static/tests/tours/utils/pos_sale_utils.js:2]: remove

**Comment by shag-odoo** [comment]
Hello @vlst-odoo 
Changes have been done
Could you please have a look

Thanks and have a great day ahead :innocent:

**Review by vlst-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**